internal/ethapi/transaction_args.go

```diff
@@ -53,6 +54,16 @@ type TransactionArgs struct {
 	// Introduced by AccessListTxType transaction.
 	AccessList *types.AccessList `json:"accessList,omitempty"`
 	ChainID    *hexutil.Big      `json:"chainId,omitempty"`
+
+	// ALEXF fields added for Type 4 AA Transaction
+	Sender        *common.Address `json:"sender"`
+	Signature     *hexutil.Bytes
+	PaymasterData *hexutil.Bytes `json:"paymasterData"`
+	DeployerData  *hexutil.Bytes
+	BuilderFee    *hexutil.Big
+	ValidationGas *hexutil.Uint64
+	PaymasterGas  *hexutil.Uint64
+	BigNonce      *hexutil.Big // AA nonce is 256 bits wide
```

Comment:
> why not change the Nonce field? duplicates are bad..

I am trying to keep the changes as much "addition only" as possible. Not here specifically, but `Nonce` is passed around as `Uint64` all over the codebase, so changing it will be tedious. We will need to do it at some point.